src/libs/AxelarAssetTransfer.ts

```diff
@@ -266,6 +294,104 @@ export class AxelarAssetTransfer {
     return address.toLowerCase();
   }
 
+  /**
+   * @param {Object}  requestParams
+   * @param {string}  requestParams.fromChain - Source chain identifier eg: avalanche, moonbeam ethereum-2, terra-2 ...
+   * @param {string}  requestParams.toChain - Destination chain identifier eg: avalanche, moonbeam ethereum-2, terra-2 ...
+   * @param {string}  requestParams.destinationAddress - Address where the asset should be transferred to on the destination chain
+   * @param {Object}  requestParams.options
+   */
+  public async sendToken(requestParams: SendTokenParams) {
+    const { fromChain, toChain } = requestParams;
+
+    await throwIfInvalidChainIds([fromChain, toChain], this.environment);
+
+    const chainList: ChainInfo[] = await loadChains({ environment: this.environment });
+    const srcChainInfo = chainList.find((ch) => ch.id === fromChain.toLowerCase());
+    if (!srcChainInfo) throw new Error("cannot find chain" + fromChain);
+
+    return srcChainInfo.module === "evm"
+      ? this.sendTokenFromEvmChain(requestParams)
+      : this.sendTokenFromCosmosChain(requestParams, srcChainInfo);
+  }
+
+  private async sendTokenFromEvmChain(requestParams: SendTokenParams) {
+    //todo
+  }
+
+  private async sendTokenFromCosmosChain(requestParams: SendTokenParams, chain: ChainInfo) {
+    if (!requestParams.options?.cosmosSendTokenOptions) throw `need a cosmos signer`;
+    const {
+      fromChain,
+      toChain: destination_chain,
+      destinationAddress: destination_address,
+      options: {
+        cosmosSendTokenOptions: {
+          cosmosDirectSigner,
+          rpcUrl,
+          transferAmount,
+          timeoutHeight,
+          timeoutTimestamp,
+          fee,
+        },
+      },
+    } = requestParams;
+    if (fromChain === CHAINS[this.environment.toUpperCase() as "TESTNET" | "MAINNET"].AXELAR)
```

Comment:
Maybe we should store a reference to `CHAINS[this.environment.toUpperCase() as "TESTNET" | "MAINNET"]` when the SDK is initialized for convenience?